go-controller/pkg/ovn/policy.go

```diff
@@ -904,6 +902,18 @@ func (oc *Controller) addNetworkPolicy(policy *knet.NetworkPolicy) {
 				handler.gress, np)
 		}
 	}
+
+	// Finally, make sure that all ACLs are set
+	np.Lock()
+	if !np.deleted {
+		for _, gp := range np.ingressPolicies {
```

Comment:
Yes, but this can only be done nicely with go-ovn or libovsdb right? We are using nbctl right now for that. I think it is a big change for this PR, maybe a different PR? Or, maybe this will be taken on by @squeed if he is working on refactoring this? Or maybe we can do this as part of the libovsdb effort?

Reply:
yeah we can do it as a separate PR